Some documentation tweaks (#1123)
JuliaDocs/Documenter.jl: docs/src/man/hosting/walkthrough.md

First, we need to Base64 encode the private key. Open Julia, and run the command

```julia

> julia> using Base64

Reviewer: ```suggestion
julia> using Base64

```

Author: Is this just a new line after the first command?

Reviewer 2: Yep. This brings it in line with how it would actually be displayed in the REPL.

Now, add a new environment variable called `DOCUMENTER_KEY`, and set its value to the output
from the Julia command above (make sure to remove the surrounding quotes).

> Finally, leave the "Display value in build log" as switched off and then click "Add".

Reviewer: ```suggestion
Finally, make sure that the "Display value in build log" is switched off and then click "Add".
```

Reviewer 2: ```suggestion
Finally, make sure that the "Display value in build log" is left switched off and then click "Add".
```

> Finally, leave the "Display value in build log" as switched off and then click "Add".

Reviewer 2: ```suggestion
Finally, leave the "Display value in build log" switched off and then click "Add".
```
